lib/datasource/aws-machine-image/index.ts

```diff
@@ -52,16 +52,17 @@ export class AwsMachineImageDataSource extends Datasource {
     });
     const matchingImages = await this.ec2.getValue().send(cmd);
     matchingImages.Images = matchingImages.Images ?? [];
-    return matchingImages.Images.sort(
-      (image1, image2) =>
-        Date.parse(image1.CreationDate) - Date.parse(image2.CreationDate)
-    );
+    return matchingImages.Images.sort((image1, image2) => {
+      const ts1 = image1.CreationDate ? Date.parse(image1.CreationDate) : 0;
+      const ts2 = image2.CreationDate ? Date.parse(image2.CreationDate) : 0;
+      return ts1 - ts2;
+    });
   }
 
   @cache({
     namespace: `datasource-${AwsMachineImageDataSource.id}`,
-    key: ({ registryUrl, lookupName }: GetReleasesConfig, newValue: string) =>
-      `getDigest:${registryUrl}:${lookupName}:${newValue ?? ''}`,
```

Comment:
`registryUrl` always is `undefined` for this datasource, isn't it?

Reply:
Yes I think so

Reply:
👍